WalletWasabi.Fluent/Models/UI/UIContext.cs

```diff
@@ -0,0 +1,18 @@
+using Avalonia;
+using Avalonia.Input.Platform;
+
+namespace WalletWasabi.Fluent.Models.UI;
+
+public class UIContext
```

Comment:
nit: rename it to `UiContext` to satisfy the naming rule plus match the convention.

Reply:
According to the convention, in two-letter acronyms, both letters must be capitalized.

![image](https://user-images.githubusercontent.com/3109851/228500738-0ce3df4a-18a5-47b0-a056-03a27f0ff7cc.png)

Source: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Reply:
https://github.com/zkSNACKs/WalletWasabi/blob/master/.editorconfig#L192

```
# name most members using PascalCase (except interface, field, local, parameter, type_parameter)
dotnet_naming_rule.most_members_must_be_pascal_case.severity = error
dotnet_naming_rule.most_members_must_be_pascal_case.symbols  = most_members_symbols
dotnet_naming_rule.most_members_must_be_pascal_case.style    = pascal_case_style

dotnet_naming_symbols.most_members_symbols.applicable_kinds   = class, struct, enum, property, method, event, namespace, local_function
dotnet_naming_symbols.most_members_symbols.applicable_accessibilities = *
```

![image](https://user-images.githubusercontent.com/16364053/228507917-9e958ae3-d97f-4379-94b7-e0f5e0be6680.png)

Similar classes in the code base:

![image](https://user-images.githubusercontent.com/16364053/228505406-a0671ff8-d940-478c-bec0-a477140098cd.png)

If we want to change to convention it should be done software-wide. I wouldn't do it in this PR.
@yahiheb opinion?

Reply:
A few points here:
- The convention I shared is quite extended. I think we should adhere to it.
- It's a minor change, but we should tackle it in a different PR.

If everything is fine with this, we can create a different PR with that renames the existing classes.

I can do that if we all agree. I'm awaiting for the team to speak 😁

Reply:
I agree with the proposed concept, but we should not differ from the current rules in any PR. I would modify the name to `UiContext` and fix the names everywhere in a following PR, once the request is discussed and approved.

Reply:
For now, it's `UiContext`.

We can change that if the team agrees. 

cc @soosr